easybuild/easyconfigs/s/SIRIUS/SIRIUS-5.5-CrayIntel-17.08.eb

```diff
@@ -0,0 +1,46 @@
+# created by Anton Kozhevnikov (CSCS)
+easyblock = 'MakeCp'
+
+name = 'SIRIUS'
+version = '5.5'
+
+homepage = 'https://electronic-structure.github.io/SIRIUS/'
+description = """Domain specific library for electronic structure calculations."""
+
+toolchain = {'name': 'CrayIntel', 'version': '17.08'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['https://github.com/electronic-structure/SIRIUS/archive/']
+sources = ['v%(version)s.tar.gz']
+
+patches = [('platform.Intel.json', './')]
+
+dependencies = [
+  ('GSL', '2.4'),
+  ('libxc', '3.0.0'),
+  ('spglib','1.10.3'),
+  ('cray-hdf5/1.10.0.3', EXTERNAL_MODULE),
+  ('cudatoolkit/8.0.61_2.4.3-6.0.4.0_3.1__gb475d12', EXTERNAL_MODULE),
+  ('magma', '2.3.0', '-cuda-8.0')
```

Comment:
If you define `cudaversion` on top of the file, you could use to define the dependency version without hard-coding it

Reply:
>4.3-6.0.4.0_3.1__gb475d12 this thing is still hardcoded